packages/yuudachi/src/commands/moderation/sub/anti-raid-nuke/coreCommand.ts

```diff
@@ -8,12 +9,17 @@ import {
 	type GuildMember,
 	type ModalSubmitInteraction,
 	type Snowflake,
+	ActionRowData,
+	ButtonComponent,
+	Message,
```

Comment:
```suggestion
	type ActionRowData,
	type ButtonComponent,
	type Message,
```

Reply:
Yeah sorry for the type imports 😅, I will look closely after them next time